added calculator.js file

kzvm opened the pull request:
I'm adding a js file with math operations for lesson02 assignment

MadEste commented on calculator.js:
```diff
@@ -0,0 +1,10 @@
+console.log(k);
+console.log(k + y);
+console.log(k / x);
+    
```
👍

MadEste commented on countdown.js:
```diff
@@ -0,0 +1,6 @@
+while (x>-1) {
+  console.log(x);
+  x--;
+}
```
👍

MadEste commented on exercise-two.js:
```diff
@@ -0,0 +1,7 @@
+let surf = ['I', 'love', 'surfing', 'in', 'my', 'home', 'town'];
+
```
I think you had some misunderstanding of what the assignment required. You should have created sentence string and converted it to an array:
`let surf = 'I love surfing in my home town.';`
`let surfArray = surf.split(' ');`

kzvm commented:
I think, I was able to fix the error, thanks

https://github.com/kzvm/html200-coding

https://github.com/UWFront-End-Cert/html200-coding/pull/33


let surf = 'I love surfing in my home town.';

const surfing = surf.split(' ');

console.log(surfing);
console.log(surfing[2]);



From: Estevan [mailto:notifications@github.com]
Sent: Monday, April 27, 2020 11:34 PM
To: UWFront-End-Cert/html200-coding <html200-coding@noreply.github.com>
Cc: Vargas (US), Kerwin Z <kerwin.z.vargas@boeing.com>; Author <author@noreply.github.com>
Subject: Re: [UWFront-End-Cert/html200-coding] added calculator.js file (#33)


@MadEste commented on this pull request.

________________________________

In exercise-two.js<https://github.com/UWFront-End-Cert/html200-coding/pull/33#discussion_r416301361>:

> @@ -0,0 +1,7 @@

+let surf = ['I', 'love', 'surfing', 'in', 'my', 'home', 'town'];

+

I think you had some misunderstanding of what the assignment required. You should have created sentence string and converted it to an array:
let surf = 'I love surfing in my home town.';
let surfArray = surf.split(' ');

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/UWFront-End-Cert/html200-coding/pull/33#pullrequestreview-401474110>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMIZL3XSNGOUJ4TXKTGCTZTROZFDRANCNFSM4MBMQIHQ>.

MadEste commented on lesson4Assig1.js:
```diff
@@ -0,0 +1,9 @@
+function numberSquared() {
+    let input = prompt('Enter a number to be squared')
```
Nice work! You might also want to use the Number() function around prompt to make sure the input is stored as a number and not a string.

MadEste commented on lesson5Assig1.js:
```diff
@@ -0,0 +1,36 @@
+function typeAction(action) {
+    let input = prompt('Type Q, W, D or B');
+
+    while (input !== 'Q' || input !== 'q') {
```
Good job checking for upper and lowercase

MadEste commented on lesson5Assig1.js:
```diff
@@ -0,0 +1,36 @@
+const Q = 'Q';
```
These const variables are not required as you do not use them elsewhere in the code.
